### DISCUSSION TITLE
Could not locate vmlinuz and initrd.img in /srv/ltsp/images/x86_64.img

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#376] [Q&A]
HI All,

I am installing LTSP on Xubuntu and after building an image I am getting the following error:

Running: ltsp kernel /srv/ltsp/images/x86_64.img
Running: mount -t tmpfs -o mode=0755 tmpfs /tmp/tmp.11hCTeChvu/tmpfs
Running: mount -t squashfs -o ro /srv/ltsp/images/x86_64.img /tmp/tmp.11hCTeChvu/tmpfs/0/looproot
Running: mount -t overlay -o upperdir=/tmp/tmp.11hCTeChvu/tmpfs/0/up,lowerdir=/tmp/tmp.11hCTeChvu/tmpfs/0/looproot,workdir=/tmp/tmp.11hCTeChvu/tmpfs/0/work /tmp/tmp.11hCTeChvu/tmpfs /tmp/tmp.11hCTeChvu/root/
Could not locate vmlinuz and initrd.img in /srv/ltsp/images/x86_64.img
root@ltspserver:~#

**Comment by alkisg** [comment]
Hi, is this because of a separate boot partition?
https://github.com/ltsp/ltsp/issues/43

**Reply by tichzvidzayi** [reply]
Yes, Here is the  df -h output 

Filesystem                         Size  Used Avail Use% Mounted on
udev                               1.9G     0  1.9G   0% /dev
tmpfs                              391M  1.5M  390M   1% /run
/dev/mapper/ubuntu--vg-ubuntu--lv  196G   10G  176G   6% /
tmpfs                              2.0G     0  2.0G   0% /dev/shm
tmpfs                              5.0M     0  5.0M   0% /run/lock
tmpfs                              2.0G     0  2.0G   0% /sys/fs/cgroup
/dev/sdb2                          976M  110M  800M  13% /boot
/dev/sdb1                          511M  7.8M  504M   2% /boot/efi
tmpfs                              391M   36K  391M   1% /run/user/1000

**Comment by alkisg** [comment]
Ok, see the solution that I linked in my previous reply.

**Reply by tichzvidzayi** [reply]
thanks it worked
